log4j2.xml

```diff
@@ -16,7 +16,7 @@
     </Appenders>
 
     <Loggers>
-        <Root level="WARN">
+        <Root level="DEBUG">
```

Comment:
should we keep it at WARN ?

Reply:
I think INFO is better (because there are no logs at all for successful case), but will revert in this one and take a look in another PR